tests/components/dsmr/test_sensor.py

```diff
@@ -98,6 +100,16 @@ async def test_default_setup(hass, dsmr_connection_fixture):
     await hass.config_entries.async_setup(mock_entry.entry_id)
     await hass.async_block_till_done()
 
+    registry = await hass.helpers.entity_registry.async_get_registry()
+
+    entry = registry.async_get("sensor.power_consumption")
+    assert entry
+    assert entry.unique_id == "1234_Power_Consumption"
+
+    entry = registry.async_get("sensor.gas_consumption")
+    assert entry
+    assert entry.unique_id == "5678_Gas_Consumption"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6372789)